conjur/ssl_service.py

```diff
@@ -7,10 +7,17 @@
 """
 # Third party
 import logging
+import os
 import socket
 from OpenSSL import SSL
 from OpenSSL.crypto import FILETYPE_PEM, dump_certificate
 
+_tls_methods = {
```

Comment:
_tls_methods-> _TLS_METHODS
what do you think?
not sure what is our convention on this. but I assume this is a constant

Reply:
tried to be uniform with the pyopenssl library I was using

<img width="400" alt="Screen Shot 2021-01-28 at 3 29 19 PM" src="https://user-images.githubusercontent.com/19418506/106145102-cba00400-617d-11eb-9c7c-0d8066f5cf43.png">